<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>BINF630_hw2</title>
    <style type="text/css">
.auto-style5 {
	text-decoration: underline;
}
.auto-style6 {
	text-align: left;
}
.auto-style7 {
	color: #0000FF;
}
.auto-style9 {
	color: #008000;
}
.auto-style10 {
	color: rgb(255, 0, 0);
}
.auto-style1 {
	color: #E8BD2E;
	}
.auto-style12 {
	color: #800080;
}
.auto-style15 {
	color: #000000;
}
.auto-style16 {
	color: #FF00FF;
}
.auto-style17 {
	font-weight: normal;
}
.auto-style18 {
	margin-left: 80px;
}
</style>
    </head>
  <body>
    <div class="auto-style6">
		<h2><strong>BINF630 Homework 2</strong><br>
        </h2>
		<h3 class="auto-style17">
        <em>Chih Hao Huang</em></h3>
		<br>
      <br>
      <div align="left">
      	<h3>
		  <span class="auto-style5"><strong>A. Basic Information</strong></span></h3>
		  <br>Protein ID assigned: <a
        href="https://www.ncbi.nlm.nih.gov/protein/380688664">
		  H8FDZ3</a><br>
		  Protein: kumamolisin<br>
          Organism: Xanthomonas citri pv. mangiferaeindicae LMG 941<br>
          GenBank ID: 
		  <a href="https://www.ncbi.nlm.nih.gov/protein/CCG36580.1">
		  CCG36580.1</a><br>
		  PUBMED: 
		  <a href="https://www.ncbi.nlm.nih.gov/pubmed/22582385">
		  22582385</a><br>
          <a href="H8FDZ3.fasta">FASTA file</a><br><br>
		  <p><span class="auto-style6">1. To collect information for protein ID <a
        href="https://www.ncbi.nlm.nih.gov/protein/380688664">
		  H8FDZ3</a>. Go to </span>
		  <a href="https://www.ncbi.nlm.nih.gov/protein/">
		  <span class="auto-style6">NCBI 
Protein website</span></a><span class="auto-style6"> and search H8FDZ3</span></p>
		  <p>
		  <img alt="" height="501" src="images/1.jpg" width="1339"></p>
		  <p><span class="auto-style6">2. After pressing "Search" button, it will take you 
directly to </span><a href="https://www.ncbi.nlm.nih.gov/protein/380688664">
		  <span class="auto-style6">this website</span></a></p>
		  <p>
		  <img alt="" height="724" src="images/3.jpg" width="1330"></p>
		  <p><span class="auto-style6">3. From here I found the </span>
		  <span class="auto-style1">protein</span><span class="auto-style6">, 
		  </span><span class="auto-style9">organism</span><span class="auto-style6">, 
		  </span><span class="auto-style7">GenBank ID</span><span class="auto-style6">, 
		  </span><span class="auto-style12">PubMed ID</span><span class="auto-style6">, and 
		  </span><span class="auto-style10">FASTA file</span><span class="auto-style6">.</span></p>
		  <p>
		  &nbsp;</p>
		  <p>
		  &nbsp;</p>
      </div>
    </div>
	<h3><strong>B. Secondary Structure:</strong></h3>
	<p><strong><a href="http://www.compbio.dundee.ac.uk/jpred/">Jpred 4</a> </strong>
	was used to predict secondary structure of assigned protein.</p>
	<p>1. Go to <a href="http://www.compbio.dundee.ac.uk/jpred/">Jpred 4</a> and 
	paste in the FASTA sequence</p>
	<p>
	<img alt="" height="817" src="images/4.jpg" width="1241"></p>
	<p>2. Click "Make Prediction" button without making any changes to the 
	settings. Result shown below:</p>
	<p>
	<img alt="" height="1282" src="images/5.jpg" width="1140"></p>
	<p>3. Now click on the "continue" button, and this is the
	<a href="http://www.compbio.dundee.ac.uk/jpred4/results/jp_sMtT__Q/jp_sMtT__Q.results.html">
	result</a> that you will see after the analysis is done:</p>
	<p>
	<img alt="" height="1098" src="images/6.jpg" width="1147"></p>
	<p>&nbsp;Part of the result shown below, full image available
	<a href="http://www.compbio.dundee.ac.uk/jpred4/results/jp_sMtT__Q/jp_sMtT__Q.svg.html">
	here</a>:</p>
	<p>
	<img alt="" height="303" src="images/7.jpg" width="1434"></p>
	<p>The<span class="auto-style7"> first line is our sequence</span>, and the
	<span class="auto-style9">green arrows in the figure indicate </span>
	<span class="auto-style9" style="font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">
	α-helixes</span><span style="color: rgb(51, 51, 51); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"> 
	and the </span>
	<span class="auto-style10" style="font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">
	red bars indicate β-sheets</span><span style="color: rgb(51, 51, 51); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">.</span></p>
	<p>&nbsp;</p>
	<h3><strong>C. Secondary Structure and 3D Structure:</strong></h3>
	<p><a href="https://swissmodel.expasy.org/"><strong>Swiss Model</strong></a> 
	was used to generate secondary structure in 3D using homology modeling 
	algorithm.</p>
	<p>1. Go to the <a href="https://swissmodel.expasy.org/">Swiss Model</a> 
	Website and click on "Start Modelling"</p>
	<p>
	<img alt="" height="175" src="images/8.jpg" width="1457"></p>
	<p>2. Paste the FASTA file into the "Target Seuqence" box, add a project 
	title (in this case, I used the protein ID), add my E-mail and click on 
	"Build Model" button (all the other settings stay as default).</p>
	<p>
	<img alt="" height="529" src="images/9.jpg" width="1499"></p>
	<p>3. <a href="https://swissmodel.expasy.org/interactive/KCA6Hk/models/">
	Result</a> show as below:</p>
	<p>
	<img alt="" height="1282" src="images/10.jpg" width="1530"></p>
	<p>Two models were generated from Swiss model, and the model is displaying&nbsp; 
	on the right (in the screenshot it's the first model). </p>
	<p>&nbsp;</p>
	<p>4. PDB models can be loaded in<a href="https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html"> 
	iCn3D</a>. Once loaded, go to 
	"Color" tab, and change to secondary, sheet in green. </p>
	<p class="auto-style18">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; First model:
	<a href="https://swissmodel.expasy.org/interactive/KCA6Hk/models/01.pdb">
	PDB</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Second model:
	<a href="https://swissmodel.expasy.org/interactive/KCA6Hk/models/02.pdb">
	PDB</a></p>
	<p><img alt="" height="602" src="images/11.jpg" width="753"><img alt="" height="511" src="images/12.jpg" width="550"></p>
	<p><span class="auto-style15">*</span><span class="auto-style9"> Green arrows in the figures above indicate
	</span>
	<span class="auto-style9" style="font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">
	α-helixes</span><span style="color: rgb(51, 51, 51); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"> 
	and the </span>
	<span class="auto-style10" style="font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">
	red sheets indicate β-sheets.</span></p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<h3>D. Quality of the Model:</h3>
	<p><a href="http://servicesn.mbi.ucla.edu/PROCHECK/"><strong>PROCHECK</strong></a> was used to check the stereochemical quality of our protein 
	structures generated by Swiss-Model by analyzing their residue-by-residue 
	geometry and overall structure geometry.</p>
	<p>1. Go to <a href="http://servicesn.mbi.ucla.edu/PROCHECK/">PROCHECK</a> 
	and upload models given above (one at a time, will use model 1 as example)</p>
	<p><img alt="" height="468" src="images/13.jpg" width="1300"></p>
	<p>2. Once done, <a href="http://servicesn.mbi.ucla.edu/PROCHECK/?job=48885">
	result page (for model 1)</a> like this will show up:</p>
	<p><img alt="" height="1212" src="images/14.jpg" width="1044"></p>
	<p>* <a href="http://servicesn.mbi.ucla.edu/PROCHECK/?job=48886">result page 
	for model 2</a></p>
	<p>3. First we will look at the "<a href="http://servicesn.mbi.ucla.edu/PROCHECK/Jobs/5219202/5219202_01.pdf">Main 
	Ramachandran plot</a>" in the result page, once click on it, this will show 
	up:</p>
	<p><img alt="" height="1128" src="images/15.jpg" width="800"></p>
	<p>From the image above, we see red, yellow, light yellow and white regions, 
	these regions indicates favored region, additional allowed region, generally 
	allowed region, and disallowed region, respectfully. As shown under the 
	figure, my model 1 has 1 residue in the disallowed regions, and 88.1% of the 
	residues inside the favored region, which indicates that model 1 has really 
	good quality. In
	<a href="https://servicesn.mbi.ucla.edu/PROCHECK/Jobs/4812707/4812707_01.pdf">
	model 2 Main Ramachandran plot</a>, there are 3 residues in the disallowed 
	region with 82% residues in the favor region. Model 2's quality is fair, but 
	model 1 is better.</p>
	<p>&nbsp;</p>
	<p>4. Next we will look at the "<a href="http://servicesn.mbi.ucla.edu/PROCHECK/Jobs/5219202/5219202_04.pdf">Main-chain 
	parameters</a>" and the "<a href="http://servicesn.mbi.ucla.edu/PROCHECK/Jobs/5219202/5219202_05.pdf">Side-chain 
	parameters</a>" (model 1)</p>
	<p><img alt="" height="1000" src="images/16.jpg" width="708"><img alt="" height="1002" src="images/17.jpg" width="706"></p>
	<p>The six parameters that we are testing in Main-chain parameters are
	<span class="auto-style10">%-tage residues in A, B, L</span>,
	<span class="auto-style7">Omega angle st dev, </span>
	<span class="auto-style9">Bad contacts / 100 residues</span><span class="auto-style15">,</span><span class="auto-style9">
	</span><span class="auto-style1">Zeta angle st dev</span><span class="auto-style15">,
	</span><span class="auto-style12">H-bond energy st dev</span>, and
	<span class="auto-style16">Overall G-factor</span>. The five parameters 
	tested in the Side-chain parameters are <span class="auto-style10">Chi-1 
	gauche minus st dev</span>, <span class="auto-style7">Chi-1 trans st dev</span>,
	<span class="auto-style9">Chi-1 gauche plus st dev</span>,
	<span class="auto-style1">Chi-1 pooled st dev</span>, and
	<span class="auto-style12">Chi-1 trans st dev</span>. In these two analysis, 
	the purple band indicates the expected parameter values, and the black line 
	inside the region is the perfect fit. When our black squares lay inside the 
	purple region, it indicates that the values are expected, the closer to the 
	perfect fit line the better the quality. In model 1 main-chain parameters 
	analysis, all six black squares lays inside the expected region; on the 
	other hand, only <span class="auto-style10">Chi-1 gauche minus st dev </span>
	<span class="auto-style15">and</span> <span class="auto-style7">Chi-1 trans 
	st dev'</span><span class="auto-style15">s value layes in the expected 
	regions, <span class="auto-style9">Chi-1 gauche plus st dev</span>,
	<span class="auto-style1">Chi-1 pooled st dev</span>, and
	<span class="auto-style12">Chi-1 trans st dev </span>parameter values are 
	not as expected. The main-chain has a good quality while the side-chain 
	quality isn't that good. </span></p>
	<p>&nbsp;</p>
	<h3>E. Structural Classification of Model:</h3>
	<p><strong><a href="http://www.cathdb.info/">CATH-Gene3D</a></strong>. CATH 
	is a classification of protein structures downloaded from the Protein Data 
	Bank and Gene3D uses the information in CATH to predict the locations of 
	structural domains on millions of protein sequences available in public 
	databases.</p>
	<p>1. Go to <a href="http://www.cathdb.info/">CATH-Gene3D</a> and search for 
	the protine "kumamolisin".</p>
	<p><img alt="" height="546" src="images/18.jpg" width="1451"></p>
	<p>2. On the result page</p>
	<p>&nbsp;</p>
	<p>3. This will take you to the next step "Submit scan". Click on "View 11 
	sequence hits now"</p>
	<p>&nbsp;</p>
	<p>4. </p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	<p>&nbsp;</p>
	</body>
</html>
